### PR TITLE
fix(token manager): Fix rounding error in IAM token manager

### PR DESCRIPTION
**Pull request by dubee**
Fixes https://github.com/IBM/go-sdk-core/issues/14

### CODE REVIEW

**Comment by dubee** [comment]
A unit test is non-trivial here due `expireTime` and `timeToLive` being initialized by a valid token claim. If there is a really a need to test this, I can move the equation to its own function in order to easily unit test the changes.

**Comment by ehdsouza** [comment]
Hi @dubee,

thanks for the PR! Could you change your commit message to follow angular commit guidelines: https://github.com/IBM/go-sdk-core/blob/master/CONTRIBUTING.md. We use semantic release, and would need a proper message for a release. Maybe: `fix(token manager):  Fix rounding error in IAM token manager`

**Comment by dubee** [comment]
@ehdsouza, I've updated the commit message and added a unit test.

File: core/iam_token_manager.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ const (
 	DEFAULT_CONTENT_TYPE        = "application/x-www-form-urlencoded"
 	REQUEST_TOKEN_GRANT_TYPE    = "urn:ibm:params:oauth:grant-type:apikey"
 	REQUEST_TOKEN_RESPONSE_TYPE = "cloud_iam"
+	fractionOfTimeToLive        = 0.8
```
Comment (ehdsouza) [inline core/iam_token_manager.go]: Since all the previous constants are in capital, let's update it as well

**Comment by dubee** [comment]
@ehdsouza, I updated the name of the constant.

**Pull request merged**